scripts/lib/publishing.ts

```diff
@@ -50,6 +50,10 @@ export function setAllVersions(version: string) {
 			path.join(packageDetails.dir, 'package.json'),
 			JSON.stringify(manifest, null, '\t') + '\n'
 		)
+		if (manifest.name === '@tldraw/ui' || manifest.name === '@tldraw/editor') {
```

Comment:
Unpkg uses the following url scheme for serving assets `unpkg.com/:package@:version/:file`. As you can see it includes the version number in the url. This is great since it means we can't cause an issue with loading assets with a different version than our editor expects.

On the other hand it does mean that we have to point the urls dynamically, based on the package version. At first I tried to just read the version from the `package.json`, but ran into all of the issues described [here](https://stackoverflow.com/questions/55753163/package-json-is-not-under-rootdir/61467483). [This solution](https://stackoverflow.com/a/69820373/1810018) did work, but it pulled in the whole `package.json`, which we probably don't want.  

So I opted into generating `version.ts` files when packaging. For now we just create them for the `ui` and `editor` packages. 

Looks like we call this when publishing a regular (alpha for now) release, as well as for canary releases, but might be good if @SomeHats or @ds300 would have a look. I'll also make sure to check if everything went right when we do the next release.

Reply:
this looks good to me